lib/logstash/codecs/cef.rb

```diff
@@ -266,13 +269,16 @@ def get_value(fieldname, event)
 
     return nil if val.nil?
 
+    key = "#{sanitize_extension_key(fieldname)}"
+    key = "#{REVERSE_MAPPINGS[key] || key}"
```

Comment:
same here, no need for string interpolation:
```rb
key = REVERSE_MAPPINGS[key] || key
```

Reply:
Ok. Will adjust my pull request.